doc/nrf/libraries/networking/lwm2m_location_assistance.rst

```diff
@@ -41,6 +41,74 @@ There are three different supported methods of obtaining the location assistance
+
+When requesting for A-GPS assistance data, the device should first set the mask for the data it is requesting by calling :c:func:`location_assistance_agps_set_mask`.
+After the mask has been set, :c:func:`location_assistance_agps_request_send` sends the request with all the necessary data to the server and responds with the A-GPS assistance data.
+The assistance data is written to the A-GPS module automatically by the library.
+
+Filtered A-GPS
+--------------
+
+There is also an option to use filtered A-GPS. In it, the satellites below the given angle above the ground will be filtered out.
+The angle can be set with :c:func:`location_assist_agps_set_elevation_mask` to a degree between `[0, 90]`. Setting the degree to -1 disables the filtering and is used by default.
+
+P-GPS location
+==============
+
+When using P-GPS location, the device requests predictions for the satellites for a near future.
+P-GPS doesn't use the information about current cell at all, it store the information about satellites and injects the data to GNSS-module when needed.
+External flash is needed when using P-GPS as the predictions needs 2kB of memory for each prediction.
+
+When requesting for P-GPS assistance data, the device can set the P-GPS resources if needed.
+If using default values in the resources, the request will be for predictions for 1 week of time. (42 predictions, 7 days, 4 hours between predictions)
+After setting the resources, :c:func:`location_assistance_pgps_request_send` sends the request to the server.
+
+Result codes and automatic resend
+=================================
+
+The location assistance objects both have a resource called `result_code`. This resource contains information about the request handling in the server side.
+It can have 3 different values:
+
+* 0  - The request was handled succesfully.
+* -1 - There is a permanent error in the server that needs fixing. Library will reject further requests and the device should be rebooted after the issue has been resolved in the server.
+* 1  - There is a temporary error in the server and device should try again after a while.
+
+The library has a resend handler for the temporary error code. It can be initialized with :c:func:`location_assistance_init_resend_handler`.
+It uses an exponential backoff for scheduling the resends. User can also decide to not use this approach and set own callbacks for the result code for better control of the application.
+
+Using A-GPS and P-GPS simultaneously
+====================================
+
+A-GPS and P-GPS can be used simultaneously. One needs to note that only one request can be ongoing at a time for the object.
+The :c:func:`location_assistance_agps_set_mask`, :c:func:`location_assistance_agps_request_send` and :c:func:`location_assistance_pgps_request_send` will return `-EAGAIN` if there is a request ongoing.
+In that case, the device should try again after the previous request has been handled.
```

Comment:
```suggestion
In that case, the device must try again after the previous request has been handled.
```
Try to do what?

Reply:
* 1  - There is a temporary error in the server and the device needs to retry sending the request after a while.